pubsub/rabbitmq/rabbitmq.go

```diff
@@ -190,10 +193,22 @@ func (r *rabbitMQ) publishSync(req *pubsub.PublishRequest) (rabbitMQChannelBroke
 		routingKey = val
 	}
 
+	ttl, ok, err := contribMetadata.TryGetTTL(req.Metadata)
+	if err != nil {
+		r.logger.Errorf("%s publishing to %s failed in TryGetTTL: %v", logMessagePrefix, req.Topic, err)
+		return r.channel, r.connectionCount, err
+	}
+	var expiration string
+	if ok {
+		// RabbitMQ expects the duration in ms
+		expiration = strconv.FormatInt(ttl.Milliseconds(), 10)
```

Comment:
This is close to what we need. Next step:
+ Write certification test to validate TTL feature.

Reply:
@Taction can you take a look into this?

Reply:
I am currently looking into how to write the certification test. Since I have not written before, it will take me some time to finish that.

Reply:
@Taction just add a test case here for TTL here:
https://github.com/dapr/components-contrib/tree/master/tests/certification/pubsub/rabbitmq

You need to create a new `component.yaml` file -- create a new subfolder in the components folder there and then reference that in your test flow.

You should verify several things:

- setting a default TTL at the component level expires messages correctly
- setting a TTL only at the message level but not component level expires messages correctly
- setting a TTL at the component level and message level ignores the default component level TTL and always uses the message level TTL specified

In all cases you should verify the message is gone after waiting the appropriate period of time.

Reply:
done